Explorer/Assets/DCL/ParcelsService/ITeleportController.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using Cysharp.Threading.Tasks;
+using DCL.AsyncLoadReporting;
 using System.Threading;
 using UnityEngine;
 
 namespace DCL.ParcelsService
 {
     public interface ITeleportController
     {
-        UniTask TeleportToSceneSpawnPointAsync(Vector2Int parcel, CancellationToken ct);
+        UniTask TeleportToSceneSpawnPointAsync(Vector2Int parcel, AsyncLoadProcessReport? loadReport, CancellationToken ct);

```

Comment:
Let's assume `AsyncLoadProcessReport` is never null to remove the corresponding checks in the implementation. Even if we don't want to pass a "real" argument we still may have a "default" instance (like Null Pattern)